### PR TITLE
Replace mentions of AppSec, Application Security to ASM

### PR DESCRIPTION
**Pull request by apigirl**
### What does this PR do?

Rebrand AppSec, Application Security to Application Security Monitoring (ASM)

### Motivation

Rebranding of the product

### Preview

https://docs-staging.datadoghq.com/arnaud.b/appsec-to-asm/security_platform/application_security/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by apigirl** [comment]
replaces https://github.com/DataDog/documentation/pull/13732

File: content/en/security_platform/application_security/getting_started/dotnet.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ You can monitor application security for .NET apps running in Docker, Kubernetes
 
    For information about which language and framework versions are supported by the library, see [Compatibility][2].
 
-2. **Enable Application Security Monitoring** by setting the `DD_APPSEC_ENABLED` environment variable to `true`. For example, on Windows self-hosted, run the following PowerShell snippet as part of your application start up script:
+2. **Enable ASM Monitoring** by setting the `DD_APPSEC_ENABLED` environment variable to `true`. For example, on Windows self-hosted, run the following PowerShell snippet as part of your application start up script:
```
Comment (kayayarai) [inline content/en/security_platform/application_security/getting_started/dotnet.md:29]:
```suggestion
2. **Enable ASM** by setting the `DD_APPSEC_ENABLED` environment variable to `true`. For example, on Windows self-hosted, run the following PowerShell snippet as part of your application start up script:
```

File: content/en/security_platform/application_security/getting_started/java.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ further_reading:
   text: 'Java Datadog Library source code'
 - link: "/security_platform/default_rules/#cat-application-security"
   tag: "Documentation"
-  text: "OOTB Application Security Rules"
+  text: "OOTB  Monitoring Rules"
```
Comment (kayayarai) [inline content/en/security_platform/application_security/getting_started/java.md:13]:
```suggestion
  text: "OOTB Application Security Monitoring Rules"
```

File: content/en/security_platform/application_security/getting_started/ruby.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ further_reading:
       text: 'Ruby Datadog Library source code'
     - link: "/security_platform/default_rules/#cat-application-security"
       tag: "Documentation"
-      text: "OOTB Application Security Rules"
+      text: "OOTB Application Security MonitoringRules"
```
Comment (kayayarai) [inline content/en/security_platform/application_security/getting_started/ruby.md:13]:
```suggestion
      text: "OOTB Application Security Monitoring Rules"
```

File: content/en/security_platform/application_security/setup_and_configure.md [hunk 4]
```diff
@@ -208,25 +208,25 @@ You can monitor application security for NodeJS apps running in Docker, Kubernet
...
 To create an exclusion filter, do one of the following:
 
-- Click on a signal in [Application Security Signals][2] and click the **Create Exclusion Filter** button in the top left corner. This method automatically generates a filter query for the targeted service.
+- Click on a signal in [ASM Signals][2] and click the **Create Exclusion Filter** button in the top left corner. This method automatically generates a filter query for the targeted service.
 - Navigate to [Exclusion Filters Configuration][3] and manually configure a new exclusion filter based on your own filter query.
 
 **Note**: Requests (traces) matching an exclusion filter are not billed.
 
-## Disable Application Security
+## Disable ASM
```
Comment (kayayarai) [inline content/en/security_platform/application_security/setup_and_configure.md:227]:
```suggestion
## Disabling Application Security Monitoring
```

File: layouts/shortcodes/appsec-getstarted.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-In general, setting up Application Security monitoring involves:
+In general, setting up Application Security Monitoring involves:
```
Comment (kayayarai) [inline layouts/shortcodes/appsec-getstarted.md:1]:
```suggestion
In general, setting up Application Security Monitoring (ASM) involves:
```

**Review by kayayarai — APPROVED** [review]

**Pull request merged**